authorreview.Rmd

```diff
@@ -1,8 +1,18 @@
-# Responding to Editor's review {#respond}
+# Responding to Editor's Review {#respond}
 
-these are placeholder headings
+```{block, type='summaryblock', echo=TRUE}
+This chapter outlines how an author of a blog post or tech note can find and respond to comments from an editor in the GitHub pull request review interface.
+```
 
-## Where to look for comments and suggested commits
+Once you have [submitted](#submitpost) your blog post or tech note, an editor will review your post providing comments on both [content](#content) (does it make sense?) and [technical](#technical) (does it follow our Style Guide?) aspects.
```

Comment:
"does it make sense?" sounds a bit harsh. "Is your story told in an optimal way?" or so?